.aspect/workflows/config.yaml

```diff
@@ -5,11 +5,20 @@ workspaces:
         label: rules_js
         tasks:
             - test:
+                  bazel:
+                        flags:
+                              - --config=aspect_rbe
+            - configure:
+                  bazel:
+                        flags: [] # TODO: Aspect CLI does not support --config for configure cmd
```

Comment:
no longer need to strip the flags since you're now applying specifically to test tasks?